src/app/base/services/project/project-apollo.service.ts

```diff
@@ -1489,4 +1490,17 @@ export class ProjectApolloService {
         })
       );
   }
+
+  getEmbeddingPlatforms(): Observable<EmbeddingPlatform> {
+    return this.apollo
+      .query({
+        query: queries.GET_EMBEDDING_PLATFORMS,
+        fetchPolicy: 'no-cache',
```

Comment:
i think the platforms wont change within a build so no need to always rerequest them. Maybe 'cache-first' ?